index.html

```diff
@@ -1,12 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <script src="js/index.js"></script>
+  <header>
+    <h1 id="title">FreeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+  </header>
+  <form id="survey-form" action="https://survey-form.freecodecamp.rocks/">
+    <div id="container">
+      <p>Name</p>
+      <input id="name" type="text" placeholder="Enter your name" required>
+      <p>Email</p>
+      <input id="email" type="email" placeholder="Enter your email" required>
+      <p>Age (optional)</p>
+      <input id="number" type="number" min="10" max="99" required>
+      <p>Which option describes your current role?</p>
+      <select name="" id="">
```

Comment:
The name and id attributes are declared with no values. It is important to do so as your markUp would be regarded as an invalid HTML. Kindly update it to ensure a functional and valid html.